Cargo.toml

```diff
@@ -1,6 +1,88 @@
+# https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#workspace-inheritance
+cargo-features = ["workspace-inheritance"]
+
 [workspace]
-members = ["contracts/*", "packages/*", "debug/*", "ci/*"]
+members = [
+	"contracts/cw-dao-core",
+	"contracts/voting-modules/*",
+	"contracts/proposal-modules/*",
+	"contracts/staking-rewards/*",
+	"contracts/utils/*",
+	"packages/*",
+	"debug/*",
+	"ci/*"
+	]
+
+[workspace.package]
+version = "2.0.0"
+edition = "2021"
+license = "Apache-2.0"
+repository = "https://github.com/DA0-DA0/dao-contracts"
+
+[workspace.dependencies]
+# Packages.
+cw-dao-core-interface = { path = "packages/cw-dao-core-interface", version = "*" }
+cw-dao-core-macros = { path = "packages/cw-dao-core-macros", version = "*" }
+cw-paginate = { path = "packages/cw-paginate", version = "*" }
+cw721-controllers = { path = "packages/cw721-controllers", version = "*" }
+indexable-hooks = { path = "packages/indexable-hooks", version = "*" }
+proposal-hooks = { path = "packages/proposal-hooks", version = "*" }
+proposal-module-testing = { path = "packages/proposal-module-testing", version = "*" }
+vote-hooks = { path = "packages/vote-hooks", version = "*" }
+voting = { path = "packages/voting", version = "*" }
+
+# Contracts.
+cw-dao-core = { path = "contracts/cw-dao-core", features = ["library"], version = "*" }
+
+# Proposal modules.
+cw-dao-proposal-single = { path = "contracts/proposal-modules/cw-dao-proposal-single", features = ["library"], version = "*" }
+cw-dao-proposal-multiple = { path = "contracts/proposal-modules/cw-dao-proposal-multiple", features = ["library"], version = "*" }
+
+# Voting modules.
+cw-dao-voting-cw20-stake = { path = "contracts/voting-modules/cw-dao-voting-cw20-stake", features = ["library"], version = "*" }
```

Comment:
`cw-dao-voting-staked-cw20`?

Reply:
oh yeah great idea. gave this that name because it uses the cw20-stake contract under the hood, but no need to leak implementation details in the name. :)